lib/src/fossacli.rs

```diff
@@ -0,0 +1,163 @@
+//! Module for FOSSA CLI container compatibility support
+//!
+//! This module enables Circe to bridge between remote OCI container formats
+//! and the tarball format expected by FOSSA CLI's container scanning system.
```

Comment:
I forget exactly, but isn't the difference that the CLI expects docker formatted tar images but many registries serve out OCI?

Reply:
it is, yes. 

i didn't want to go into too much detail in here just because as a separate codebase fossa cli may change over time causing any docs here to be incorrect.

Reply:
sounds good!